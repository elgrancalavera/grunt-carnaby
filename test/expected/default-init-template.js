/*
 * {%= name %}
 * {%= filename %}
 * {%= repository.url %}
 * Copyright (c) {%= grunt.template.today('yyyy') %} {%= author.name %}
 * {%= author.url %}
 */
