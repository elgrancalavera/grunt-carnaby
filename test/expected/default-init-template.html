<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <title>{%= name %}</title>
    <meta name="description" content="{%= description %}">
    <meta name="viewport" content="width=device-width">
    <link rel="stylesheet" href="styles/main.css">
  </head>
  <body>
    <!--[if lt IE 7]>
      <p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
    <![endif]-->
    <section id="main"></section>
    <!-- For the sake of clarity, we are only using the HTML5BP plugins because
    of the console shim -->
    <script src="scripts/common/plugins.js"></script>
  </body>
</html>
